src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
+    #if($n.startsWith("googleapps"))#set($prf="")#end
+    #set($msg=$services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p))##
+    $stringtool.replacePattern($msg, '\[\[([^>]*)>>([^\]]*)\]\]', '<a href="$2">$1</a>')##
+#end
+#macro (displayInput $property)##
+    #set ($output = $doc.display($property, 'edit', $obj))##
+    #set ($output = $stringtool.removeStart($output, '{{html clean="false" wiki="false"}}'))##
+    #set ($output = $stringtool.removeEnd($output, '{{/html}}'))##
+    $output##
 #end
-$services.localization.render('googleapps.config.explanation')
 
+$services.localization.render('googleapps.config.explanation')
 
-{{html wiki=true}}
+{{html wiki=false}}
```

Comment:
For the `{{html}}` macro, the `wiki` parameter is set, by default, to `false`, so you don't need to use it. What would be nice instead, is  to use the other well known parameter: `clean="false"`. Please take a look at https://extensions.xwiki.org/xwiki/bin/view/Extension/HTML%20Macro#HParametersdefinition.

Reply:
I did so.
I am not sure it is best practice to add clean=false but it is certainly better for performance.